benchmark/e2e-ci/benchmark-run.js

```diff
@@ -60,7 +60,7 @@ const getWorkflowRunsInProgress = () => {
   return new Promise((resolve, reject) => {
     let response = ''
     const request = https.request(
-      `${GET_WORKFLOWS_URL}?event=workflow_dispatch&status=in_progress OR waiting OR requested OR queued`,
+      `${GET_WORKFLOWS_URL}?event=workflow_dispatch`,
```

Comment:
the status filter does not accept multiple fields, so this was not working. We were getting all the workflows